scanner/config.yaml.sample

```diff
@@ -6,11 +6,17 @@ indexer:
     conn_string: "host=/var/run/postgresql"
     password_file: ""
   get_layer_timeout: 1m
+  repo:
+    rhel-repository-scanner:
+        repo2cpe_mapping_url: https://sensor.stackrox.svc/scanner/definitions?type=repo2cpe
+  package:
+    rhel_containerscanner:
+        name2repos_mapping_url: https://sensor.stackrox.svc/scanner/definitions?type=name2cpe
```

Comment:
Does it make sense to tie the ScannerV4 config so closely to the inner workings of `claircore`?  The map keys are specific identifiers to `claircore` internals (ie: `rhel-repository-scanner`), my first thought is we shouldn't tie the two so closely - suppose that pro/cons can be debated.

Reply:
I can't think of other options to enable scanner to fetch those two files from sensor. Do you guys have any suggestions?

@jvdm @RTann

Reply:
David's comment is related to the Config struct layout. You're using a generic interface which is closer to ClairCore, but we don't have to do that. We can have our own fields in the config and in the instantiation of the indexer we grab the information from the config and pass it down to the libindexer in the format needed.

That's better because it decouples our config from ClairCore. For example: There is no need to change k8s templates if ClairCore config changes. It's also cleaner to read. Also, it is easier for us to make our own changes without conflicting with ClariCore (kinda the other way around, from my first reason). I left a suggestion on how the config could look like in another comment.